src/bin/easage_pack.rs

```diff
@@ -36,9 +38,9 @@ pub fn get_command<'a, 'b>() -> App<'a, 'b> {
                 .long(ARG_NAME_KIND)
                 .value_name(ARG_NAME_KIND)
                 .takes_value(true)
-                .required(true)
-                .possible_values(&["BIGF", "BIG4"])
-                .help("archive kind (BIGF or BIG4, case-sensitive)"))
+                .default_value(ARG_VALUE_KIND_BIGF)
+                .possible_values(&[ARG_VALUE_KIND_BIGF, ARG_VALUE_KIND_BIG4])
+                .help("FourCC to use (case-sensitive)"))
```

Comment:
Please change this to `use BIG4 for the Battle for Middle-Earth series or BIGF for Generals / Zero-Hour` (intentionally not including the `(case-sensitive)` bit which I have changed my mind about).

LGTM after that.